site2/website/data/connectors.js

```diff
@@ -5,6 +5,18 @@ module.exports = [
         type: 'Sink',
         link: 'https://www.aerospike.com/'
     },
+    {
```

Comment:
I think the list is sorted in alphabet order. so you have to add them to be after `data-generate` and before `elastic-search`.